src/helpers/performance-report-analyser.helper.js

```diff
@@ -0,0 +1,36 @@
+import fs from 'fs';
+import chalk from 'chalk';
+
+class PerformanceReportAnalyser {
+  getReport(fileName) {
```

Comment:
This is a problem with "class" in es6. All methods are added to prototype, so they are public - but we don't wont this.

There are two possible solutions for this. First is revealing module patter -> return an wrapper for PerformanceReportAnalyser that reveals only a checkTiming method, the second one is to move those methods outside class - const getReport etc.